examples/blazor/BlazingFast.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="5.0.0-rc.1.*" />
```

Comment:
RC2 just got released, maybe update it?

Reply:
I followed the instructions here: https://docs.microsoft.com/en-us/aspnet/core/migration/31-to-50?view=aspnetcore-3.1&tabs=visual-studio and hadn't realized an new RC was out. Let me see if it's a simple change of a few version numbers...

Reply:
Ok, I installed RC2 on my machine, updated the project csproj package versions and everything seems to work.